GsaGH/Components/5_Results/ReactionForceDiagrams.cs

```diff
@@ -118,10 +117,12 @@ protected override void SolveInstance(IGH_DataAccess dataAccess)
       ReadOnlyDictionary<int, Node> gsaFilteredNodes = gsaResult.Model.Model.Nodes(filteredNodes);
       ConcurrentDictionary<int, GsaNodeGoo> nodes = Helpers.Import.Nodes.GetNodeDictionary(gsaFilteredNodes, lengthUnit);
 
-      // get scale or compute autoscale
-      double scale = 1;
-      if (!dataAccess.GetData(2, ref scale))
-        scale = ComputeScale(forceValues, gsaResult.Model.BoundingBox);
+      // compute autoscale
+      double scale = ComputeScale(forceValues, gsaResult.Model.BoundingBox);
+      // optional factor
+      double factor = 1;
+      if (dataAccess.GetData(2, ref factor))
+        scale *= factor;
```

Comment:
This would make the length of the vectors non-consistent size across different model (different sizes of the model’s bounding box). 

If users input a scaling factor (say 2) shouldn’t that always make a reaction force of 5kN be draw with a length of 2*5=10 no matter the size of the model?

Reply:
@a-tiwari17 fyi

Reply:
@tlmnrnhrdt shall we close this PR?